armi/bookkeeping/db/database3.py

```diff
@@ -439,11 +441,6 @@ def loadState(
                 )
             )
 
-        if updateGlobalAssemNum:
-            updateGlobalAssemblyNum(newR)
-
-        self.o.reattach(newR, self.cs)
```

Comment:
Why did you remove the `self.o.reattach()`?

Just curious.

Reply:
I didn't remove it, I just moved it closer to where it is actually relevant.

The `else` clause of that conditional only raises errors. So having the `reattach` wait until after the `else` didn't seem necessary, and I just thought it helped with code clarity to move the call into the `if` clause.